testsuite/features/step_definitions/common_steps.rb

```diff
@@ -570,13 +570,18 @@
 When(/^I enable repositories before installing Docker$/) do
   # Distribution Pool and Update
   os_version = get_os_version($minion, false)
-  arch, _code = $minion.run('uname -m')
-  puts $minion.run("zypper mr --enable SLE-#{os_version}-#{arch.strip}-Pool")
-  puts $minion.run("zypper mr --enable SLE-#{os_version}-#{arch.strip}-Update")
+  if os_version != "15"
+    arch, _code = $minion.run('uname -m')
+    repos = "SLE-#{os_version}-#{arch.strip}-Pool SLE-#{os_version}-#{arch.strip}-Update"
+    puts $minion.run("zypper mr --enable #{repos}")
+  else
+    repos, _code = $minion.run('zypper lr | grep SLE-Module-Basesystem | cut -d"|" -f2')
+    puts $minion.run("zypper mr --enable #{repos.gsub(/\s/, ' ')}")
```

Comment:
I might be wrong as I'm not sure about the output stored in `repos` variable, but it what about using `repos.strip` instead of `repos.gsub(/\s/, ' ')`
Apart from that, I think we are not using `_code` variable, we might remove it.

Reply:
No, the problem is to remove the linefeed in the middle of the variable, not to chop ending whitespace.

If there is extra whitespace at the end of the command, it's just a linux command ending with whitespace, nothing bad will happen :smile_cat: .

`_` prefix is a convention for variables that we don't use. I find it a bit clearer and safer to mention it explicitely, but we could also remove it, yes. Are you sure about the rules for omitted variables in Ruby ? I'm not :blush: .